unity/EVPreprocessing/Assets/Editor/ModelImport/ModelImporter.cs

```diff
@@ -87,6 +87,12 @@ protected void AddLayerComponent(GameObject go, ModelLayerInfo layerInfo)
             ModelLayer layer = go.AddComponent<ModelLayer>();
             layer.Caption = layerInfo.Caption;
             layer.Simulation = CheckIfSimulation(layerInfo.DataType);
+            layer.DataType = GetDataType(layerInfo.DataType);
+        }
+
+        private DataType GetDataType(string layerDataType)
+        {
+            return layerDataType == "volumetric" ? DataType.Volumetric : DataType.Mesh;

```

Comment:
I think the naming is fine for now, but later it might get confused in the preprocessing with simulation flag as they both come from DataType field in ModelInfo.json. I left simulation flag intentionally to preserve backward-compatibility with the current shaders but later we should think about integrating these two fields.